### PR TITLE
[Refactoring] - CON 485 - Align language list requests

### PR DESCRIPTION
**Pull request by 404passionFruit**
**Jira issue**
https://oat-sa.atlassian.net/browse/CON-485

**What has changed**
I refactored itemLoader to make separate request to fetch list of languages using `request` lib from tao-core-sdk
Also updated mocks in tests for `itemData` and `languages`

**Testing environment**
http://test-sveta.playground.kitchen.it.taocloud.org:40431/

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-qti-item",
-    "version": "0.3.3",
+    "version": "0.3.4",
```
Comment (habemuscode) [inline views/package.json:3]: 1. Why bump version in package.json if not any library upgrade?
2. If needed, push `package.lock` too after `npm i` in this `views/` folder

**Review by habemuscode — CHANGES_REQUESTED** [review]
One small comment

File: views/package.json (hunk 1 above)
Comment (404passionFruit) [reply views/package.json]: Good point, you are right

**Review by habemuscode — APPROVED** [review]

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by 404passionFruit** [comment]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**